huxley/settings/main.py

```diff
@@ -30,7 +30,7 @@
 TIME_ZONE = 'America/Los_Angeles'
 USE_I18N = False
 
-EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+EMAIL_BACKEND = 'huxley.logging.LoggingEmailBackend.DevLoggingEmailBackend'
```

Comment:
We'll want to update the value in the local.py.default file (and we need to update it in our deployments also).